muni_portal/tests/integration/test_webpush.py

```diff
@@ -0,0 +1,69 @@
+from django.contrib.auth.models import User
+from django.test import Client, TestCase
+from django.urls import reverse
+from django_q.models import Success as SuccessTask
+from faker import Faker
+from rest_framework import status
+from rest_framework.test import APIClient
+from unittest.mock import patch
+
+from muni_portal.core.models import WebPushNotification, WebPushSubscription
+
+faker = Faker()
+
+
+class WebPushNotificationIntegrationTestCase(TestCase):
+
+    def setUp(self):
+        self.dj_client = Client()
```

Comment:
Why not just use self.client?

Reply:
To be sure that we don't mix up API JWT session and Django admin session.